tango/gevent_executor.py

```diff
@@ -49,18 +50,13 @@ def set_global_executor(executor):
     _EXECUTOR = executor
 
 
-# Patch for gevent threadpool
-
 def get_global_threadpool():
-    """Before gevent-1.1.0, patch the spawn method to propagate exception
-    raised in the loop to the AsyncResult.
-    """
-    threadpool = gevent.get_hub().threadpool
-    if gevent.version_info < (1, 1) and not hasattr(threadpool, '_spawn'):
-        threadpool._spawn = threadpool.spawn
-        threadpool.spawn = types.MethodType(
-            spawn, threadpool, type(threadpool))
-    return threadpool
+    global _THREAD_POOL
+    try:
+        return _THREAD_POOL
+    except NameError:
+        _THREAD_POOL = ThreadPool(10)
```

Comment:
Do we really need this limit of 10?

Reply:
No, you are right. I will make it unlimited